torchrl/collectors/collectors.py

```diff
@@ -699,7 +699,6 @@ def _step_and_maybe_reset(self) -> None:
             # collectors do not support passing other tensors than `"_reset"`
             # to `reset()`.
             if len(self.env.batch_size):
-                self._tensordict.masked_fill_(done_or_terminated, 0)
```

Comment:
What was the utility of ths?

It was assuming that the done_spec is expandable to the other key specs